libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1043,7 +1043,29 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 	}
 	case FunctionType::Kind::Error:
 	{
-		solAssert(false, "");
+		vector<string> arguments;
+		vector<Type const*> argumentTypes;
+		for (ASTPointer<Expression const> const& arg: _functionCall.sortedArguments())
+		{
+			argumentTypes.push_back(arg->annotation().type);
+			arguments += IRVariable(*arg).stackSlots();
+		}
+		ABIFunctions abi(m_context.evmVersion(), m_context.revertStrings(), m_context.functionCollector());
```

Comment:
Maybe moving it down when it is used. Also, you can inline it with using `m_context.abiFunctions()`